modules/material-management/src/needs.ts

```diff
@@ -21,3 +21,13 @@ export function needs(
     throw new Err(errorMessage)
   }
 }
+
+export class NotSupported extends Error {
+  code: string
+
+  constructor(message?: string) {
+    super(message)
+    Object.setPrototypeOf(this, NotSupported.prototype)
+    this.code = 'NOT_SUPPORTED'
+  }
+}
```

Comment:
Why does this error go here?
This is for the integration modules.

Reply:
@seebees It is now used in `modules/raw-rsa-keyring-node/src/oaep_hash_supported.ts`.  Line 35.
This change was from a call we had a long time ago when we were first working on this PR.